.github/workflows/gradle.yml

```
@@ -0,0 +1,50 @@
+        uses: actions/checkout@master
+
+      - name: Set up repository
+        uses: actions/checkout@master
+        with:
+          ref: master
+
+      - name: Merge to master
+        run: git checkout --progress --force ${{ github.sha }}
+
+      - name: Validate Gradle Wrapper
+        uses: gradle/wrapper-validation-action@v1
```

actionlint (1.7.7)

the runner of "gradle/wrapper-validation-action@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)